lib/puppet/util.rb

```diff
@@ -7,6 +7,7 @@
 require 'pathname'
 require 'ostruct'
 require_relative 'util/platform'
+require 'puppet/util/windows'
```

Comment:
We switched to require relative in main to speed up load times. Could you change this to

```suggestion
require_relative 'util/windows'
```